packages/internal/src/utils.js

```diff
@@ -381,3 +381,25 @@ export const fsStreamReady = stream =>
     stream.on('ready', onReady);
     stream.on('error', onError);
   });
+
+/**
+ * Concise way to check values are available from object literal shorthand.
+ * Throws error message to specify the missing values.
+ *
+ * @template {Record<string, unknown>} T
+ * @param {T} obj
+ * @throws if any value in the object entries is not defined
+ * @returns {asserts obj is Required<T>}
+ */
+
+export const assertDefined = obj => {
```

Comment:
I'm not crazy about this name. Maybe `assertAllDefined` or `assertFullyDefined`?

Reply:
Good point. I went with `All` because `Fully` suggests to me a recursive check.